README.md

```diff
@@ -43,15 +43,13 @@ Create a `TimeSeries` object from a Pandas DataFrame, and split it in train/vali
 
 ```python
 import pandas as pd
-from darts import TimeSeries
+from darts.datasets import AirPassengersDataset
 
-df = pd.read_csv('AirPassengers.csv', delimiter=",")
+df = AirPassengersDataset.load()
 series = TimeSeries.from_dataframe(df, 'Month', '#Passengers')
```

Comment:
```suggestion
series = AirPassengersDataset.load()
```

Reply:
In fact yesterday I started having second thoughts about changing the README to use Datasets. I'm thinking in the Readme it might be better to explicitly show how Darts can be easily used starting from a Dataframe. Otherwise users may think this dataset business is hiding some complexity.